sw/device/lib/testing/i2c_testutils.c

```diff
@@ -220,7 +222,7 @@ status_t i2c_testutils_read(const dif_i2c_t *i2c, uint8_t addr,
     TRY(dif_i2c_read_byte(i2c, data++));
   }
 
-  return OK_STATUS();
+  return OK_STATUS((int32_t)nak_count);
```

Comment:
Is this cast required because of something in the macro?

Reply:
Yes, the `status_t` wraps an `int32_t`. [See](https://cs.opensource.google/opentitan/opentitan/+/master:sw/device/lib/base/status.h;drc=1ff6f6ea02a1711224a284a8855264406750eabd;l=56
)

Reply:
Ah, thanks